enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,360 @@
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
+
+We believe this feature set is generic enough to support a wide range of initial bare metal cluster use cases, including:
+
+* support for an environment where developers have to flexibility to install their own operating systems and software
+* support for alternative clusters such as SLURM
+* support for bare metal configuration during O-SAC cluster fulfillment
+
+In this context, a “tenant” is an organization with admin rights to the requested bare metal resources; at the MOC, a tenant
+would be led by a ColdFront PI or manager.
+
+Although this proposal only covers bare metal fulfillment (and not VMs), we believe that developing this bare metal capability
+independently will allow us to create a thorough and reliable implementation that can later be used as a solid base for
+VDC fulfillment.
+
+## Motivation
+
+Bare metal fulfillment is a fundamental requirement for O-SAC, identified as an explicit need for the MOC. It will also serve
+as a component for other fulfillment workflows, such as VDC and OpenShift cluster fulfillment.
+
+### User Stories
+
+* As a provider, I want to be able to define the available bare metal resource classes.
+* As a provider, I want to mark hosts as available for bare metal fulfillment.
+* As a tenant, I want to be able to see available bare metal resource classes.
+* As a tenant, I want to be able to select hosts for a bare metal cluster by resource class and other possible filters (such as rack location).
+* As a tenant, I want to be able to increase or decrease the hosts for an existing bare metal cluster.
+* As a tenant, I want to be able to view inventory details for my hosts.
+* As a tenant, I want to be able to connect to the serial console of my hosts.
+* As a tenant, I want to be able to perform basic power control (power on/power off/reset) of my hosts.
+* As a tenant, I want to be able to attach hosts to existing networks.
+* As a tenant, I want to be able to modify the network connectivity of my hosts by attaching or detaching networks.
+* As a tenant, I want to be able to manage connectivity at the interface level in order to support hardware that may have both regular and high performance interfaces (for example, some hosts may have high-bandwidth connectivity for GPU workloads, and it may be necessary to attach specific networks to these interfaces).
+
+### Goals
+
+We will implement bare metal fulfillment in the O-SAC solution, covering the user stories described above. This implementation will
+be first deployed and tested in the MOC; as a result, the implementation will rely upon ESI for bare metal management (although
+it will be designed to allow easy replacement of the bare metal management layer). While not addressed in this proposal, a long term goal
```

Comment:
```suggestion
it will be designed to allow replacement of the bare metal management layer). While not addressed in this proposal, a long term goal
```